rust-runtime/aws-smithy-types/test_data/date_time_format_test_suite.json

```diff
@@ -0,0 +1,1334 @@
+{
+  "format_date_time": [
+    { "epoch_seconds": -377703389549, "epoch_subsecond_nanos": 1000000, "iso": "-9999-01-20T23:47:31.001Z",
```

Comment:
I'm a little confused about `formatted`: is that HTTP date? sometimes it's a negative number? Sometimes it's an http date? Can this be named something more descriptive?